pages/index.html

```diff
@@ -5,8 +5,34 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="/styles/styles.css" rel="stylesheet">
 </head>
-<body>
-  
+<body class="grid-container">
```

Comment:
It is a convention to organize the files such that `index.html` is at the project root director level, and the rest of the pages are in a `pages` directory.